### DISCUSSION TITLE
[Question] boot.netboot.xyz domain address change

### DISCUSSION THREAD

**Discussion by abdullah-erturk** [antonym/netboot.xyz#1085] [General] [answered]
**I work at a university and we already have an ipxe server of our own, I want to improve it a little.

How can I change the boot.netboot.xyz domain address with my own domain address by editing the netboot.xyz.iso file?

Many thanks for this project...**

**Comment by antonym** [comment, marked as answer]
See:

https://netboot.xyz/docs/selfhosting

You could use this to generate the files and use user_overrides.yml to override the defaults and customize the files to what you need. Moving this to discussion as it's not a bug.
